dist/ami/files/scylla_install_ami

```diff
@@ -63,15 +63,14 @@ if __name__ == '__main__':
 
     run('yum update -y')
 
-    run('curl -L -o /etc/yum.repos.d/scylla-ami-drivers.repo https://copr.fedorainfracloud.org/coprs/scylladb/scylla-ami-drivers/repo/epel-7/scylladb-scylla-ami-drivers-epel-7.repo')
     if args.repo_for_install:
         run('curl -L -o /etc/yum.repos.d/scylla_install.repo {REPO_FOR_INSTALL}'.format(REPO_FOR_INSTALL=args.repo_for_install))
 
     if args.localrpm:
-        rpms = glob.glob('/home/centos/scylla*.*.rpm')
+        rpms = glob.glob('/home/ec2-user/scylla*.*.rpm')
```

Comment:
While at it, it's a good opportunity to parameterize `$USER` and `$HOME`

Reply:
Changed the script to auto-detect home directory: https://github.com/scylladb/scylla/pull/5540/files#diff-2fe20e870f705ad288eafddbd5031182R74